assets/less/variables.less

```diff
@@ -47,4 +50,4 @@
 
 // Label styles
 @theme-label-bg: lighten(@theme-blue, 15%);
-@theme-label-border-color: darken(@theme-label-bg, 20%);
```

Comment:
I'm not too certain this looks familiar either. Odd, both files have newline issues at the end. Perhaps Grunt is responsible for these.